### PR TITLE
More TRPl editing

### PR DESCRIPTION
**Pull request by steveklabnik**
A bunch of chapters, fixes an issue or two as well.

r? @alexcrichton

### CODE REVIEW

File: src/doc/trpl/match.md [hunk 1]
```diff
@@ -20,16 +18,18 @@ match x {
...
 
-`match` takes an expression and then branches based on its value. Each *arm* of
+[if]: if.html
+
+`match` takes an expression and then branches based on its value. Each ‘arm’ of
 the branch is of the form `val => expression`. When the value matches, that arm’s
 expression will be evaluated. It’s called `match` because of the term ‘pattern
 matching’, which `match` is an implementation of. There’s an [entire section on
-patterns][patterns] coming up next, that covers all the options that fit here.
+patterns][patterns] hat covers all the patterns that are possible here.
```
Comment (mdinger) [inline src/doc/trpl/match.md]: `s/hat/that/`

Comment (steveklabnik) [inline src/doc/trpl/match.md]: thanks

**Comment by steveklabnik** [comment]
:confetti_ball:

**Pull request merged**